frontend/src/components/Tiers.js

```diff
@@ -152,6 +188,37 @@ export default class Tiers extends Component {
       </div>
     );
   }
+
+  callApplePay(tier, payload) {
+    const { collective, onToken } = this.props;
+
+    // setup the payment request
+    const paymentRequest = {
+      countryCode: 'US', // TODO: find out if we need to change this for other countries
+      currencyCode: collective.currency,
+      total: {
+        label: collective.name,
```

Comment:
shouldn't the label be something more explicit (including the `tier.description`)?